docs/setup/upgrade.asciidoc

```diff
@@ -7,6 +7,7 @@ To upgrade from 7.16.0 or earlier to {version},
 Before you upgrade, you must resolve all critical issues identified by the *Upgrade Assistant*.
 
 In addition, we recommend to carefully review the <<breaking-changes-summary,list of breaking changes and deprecations>> and to take any necessary actions to mitigate their impact on the upgrade.
+You can active the <<allow-http-debug-logs, deprecated APIs debug logs config>> to get information about calls to deprecated APIs.
```

Comment:
```suggestion
You can enable the <<logging-settings, `deprecation.allow_http_debug_logs` logging setting>> to get information about calls to deprecated APIs.
```
This will link to the logging settings page as a whole instead of the specific row, as linking to rows isn't supported. To work around it, we can specify the exact setting we're referring to.